Dockerfile.gaas-ui

```diff
@@ -0,0 +1,29 @@
+# STAGE 1 - BUILD UI APP
```

Comment:
Would it be okay if this and [Dockerfile.gaas-rest](https://github.com/gchq/gaffer-experimental/blob/99fc7cb396647798eb2fa2c968eb88cf22daca5e/Dockerfile.gaas-rest`) are moved into their respective folders?

Reply:
Sorry they shouldn't have been included as part of the commit as they are just used for the development pipeline. I have removed them.